app/views/clientsmodal.html

```diff
@@ -62,7 +62,12 @@ <h4 class="modal-title" id="addClientLabel">
 
       <div class="form-group" ng-class="{ 'has-error' : ngError.roles }">
         <label><span>*</span> Roles</label>
-        <taglist tag-data="client.roles"><input class="form-control" placeholder="Enter a role and press enter..." type="text" name="roles" ng-model="roles" typeahead="role for role in taglistClientRoleOptions | filter:$viewValue" /></taglist>
+        <br/>
+        <span ng-repeat="role in selectClientRoleOptions">
+          <input type="checkbox" ng-model="bleh" ng-checked="assigned[role]" ng-change="toggleRoleAssignedToClient(role)">{{role}}</input>
```

Comment:
Hi @rcrichton . I want the functionality of the ng-change directive, which requires an ng-model for some arbitrary reason, so I created a dummy.  Couldn't think of a better name offhand :)